libexec/alluxio-config.sh

```diff
@@ -41,6 +41,11 @@ fi
 
 . "${ALLUXIO_CONF_DIR}/alluxio-env.sh"
 
+if [[ -n "${ALLUXIO_MASTER_ADDRESS}" ]]; then
```

Comment:
@apc999 could you take a look at this to make sure that this does not interfere with your configuration changes? thanks